source/config/configSpec.py

```diff
@@ -47,7 +47,6 @@
 	delayedCharacterDescriptions = boolean(default=false)
 	excludedSpeechModes = int_list(default=list())
 	trimLeadingSilence = boolean(default=true)
-	useWASAPIForSAPI4 = featureFlag(optionsEnum="BoolFlag", behaviorOfDefault="enabled")
```

Comment:
Moving the SAPI4 setting seems to cause too many API breaking changes.
@SaschaCowley - thoughts on just making the SAPI5 setting user facing?
There are only 2 known issues with SAPI4, and at least one will be fixed in 2025.3
https://github.com/nvaccess/nvda/issues/18310

Otherwise we need to keep the combo box and in sync. I think we should instead deprecate `useWASAPIForSAPI4` for removal in 2026.1, in favour of removing the setting entirely (if we fix both synths) or surfacing it like SAPI5